### PR TITLE
Add support for XT87

### PR DESCRIPTION
**Pull request by kopibeng**
Add support for XT87

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kopibeng/xt87/info.json
Comment (waffle87) [inline keyboards/kopibeng/xt87/info.json:1]: Please run `qmk format-json path/to/info.json` on this file.

File: keyboards/kopibeng/xt87/info.json [hunk 1]
```diff
@@ -0,0 +1,888 @@
...
+		"caps_lock": "B3",
+		"scroll_lock": "A15"
+	},
+	"ws2812": {
+		"pin": "A10"
+	},
+	"rgblight": {
+		"pin": "A10",
+		"led_count": 19,
+		"hue_steps": 8,
```
Comment (waffle87) [inline keyboards/kopibeng/xt87/info.json:37]:
```suggestion
		"led_count": 19,
```

File: keyboards/kopibeng/xt87/keymaps/default/keymap.c [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+	
+	// Fn3 Layer
+	[3] = LAYOUT_all(
+		KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS
+	),
```
Comment (waffle87) [inline keyboards/kopibeng/xt87/keymaps/default/keymap.c:59]:
```suggestion
	)
```

File: keyboards/kopibeng/xt87/keymaps/via/keymap.c [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+	
+	// Fn3 Layer
+	[3] = LAYOUT_all(
+		KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS, KC_TRNS, KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,          KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, 
+		KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS,                            KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,                   KC_TRNS, KC_TRNS, KC_TRNS
+	),
```
Comment (waffle87) [inline keyboards/kopibeng/xt87/keymaps/via/keymap.c:59]:
```suggestion
	)
```

File: keyboards/kopibeng/xt87/readme.md [hunk 4]
```diff
@@ -0,0 +1,25 @@
+# XT87
+
+![XT87](https://i.imgur.com/N1QaXPTl.jpg)
```
Comment (waffle87) [inline keyboards/kopibeng/xt87/readme.md:3]:
```suggestion
![XT87](https://i.imgur.com/N1QaXPTlh.jpg)
```

**Review by waffle87 — CHANGES_REQUESTED** [review]

File: keyboards/kopibeng/xt87/readme.md (hunk 4 above)
Comment (kopibeng) [reply keyboards/kopibeng/xt87/readme.md]: Changed imgur link to huge thumbnail from large.

**Comment by kopibeng** [comment]
Done. Please let me know if there is any issue 😄

**Review by waffle87 — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**